osf/management/commands/import_EGAP.py

```diff
@@ -258,6 +258,11 @@ def main(guid, creator_username):
         anon_titles = ', '.join([data['data']['attributes']['name'] for data in anon_metadata_dict])
         registration_metadata['q38'] = {'comments': [], 'extra': anon_metadata_dict, 'value': anon_titles}
 
+        try:
+            embargo_date = registration_metadata.pop('q12')
+        except KeyError:
+            embargo_date = None
```

Comment:
nitpick: this could be just `embargo_date = registration_metadata.pop('q12', None)`